### ISSUE TITLE
Windows blocks execution of a program located in external device

### ISSUE THREAD

**Issue by treneth333** [sanscript-tech/hacking-tools-scripts#411]
When attempting to run toolbox.exe from my flash drive windows 10 blocks execution
But if I run the individual .exe files that are within the program's data folder they execute flawlessly

The only difference is that toolbox.exe is a GUI used to execute a multitude of tools.

On a security standpoint, wouldn't it be better to block the program's that could actually directly impact the PC?

Some of the program's were capable of making major changes to the system without admin permission.

Could this be utilized to run specific programs in admin permission status?

Since I stumbled upon it I've been tinkering with .bat files and .exe files 

Unsigned .exe files almost always get flagged
.bat files don't create as much of a wave

So if the .exe file only interacts with the files in the USB drive, it's less likely to be noticed.
Allowing the .exe file to run the .bat files and use them to directly interact with windows.

In conclusion what was meant to protect the OS inadvertently allowed it to be exploited.

For the white hats:
Anyone know how to fix this exploit?
